WalletWasabi/Blockchain/Blocks/BlockNotifier.cs

```diff
@@ -17,6 +17,7 @@ public BlockNotifier(TimeSpan period, IRPCClient rpcClient, P2pNode? p2pNode = n
 		RpcClient = Guard.NotNull(nameof(rpcClient), rpcClient);
 		P2pNode = p2pNode;
 		ProcessedBlocks = new List<uint256>();
+		BestBlockHash = uint256.Zero;
```

Comment:
IMO we could move these two next to their properties.
Like:
`	public uint256 BestBlockHash { get; private set; } = uint256.Zero;
`